alibi/explainers/backends/cfrl_tabular.py

```diff
@@ -506,7 +506,7 @@ def get_inv_preprocessor(X_ohe: np.ndarray):
 
         np_X_inv = np_X_inv[:, inv_perm].astype(object)
         for i, fn in enumerate(feature_names):
-            type = feature_types[fn] if fn in feature_types else float
+            type = feature_types[fn] if fn in feature_types else float  # type: ignore # TODO: closure resets type info?
```

Comment:
See here https://mypy.readthedocs.io/en/stable/common_issues.html#narrowing-and-inner-functions

Reply:
The issue here is with the inferred type for `feature_types`, `mypy` can't see that it can't be `None` due to it being inside an inner function which is a limitation referred to in the link above. I will see if I can reduce the scope of the ignore with an error code though.